SearchableTools/ViewModel.swift

```diff
@@ -33,4 +36,15 @@ class ViewModel: ObservableObject {
         }
       }
     }
+    
+    
+    func addButton() {
```

Comment:
`addButton` is not the correct name for this function because we are adding a tool instead of a button.

Let's rename this to
```suggestion
    func addNewTool() {
```